homeassistant/components/ipp/sensor.py

```diff
@@ -34,6 +34,8 @@ async def async_setup_entry(
 ) -> None:
     """Set up IPP sensor based on a config entry."""
     coordinator: IPPDataUpdateCoordinator = hass.data[DOMAIN][entry.entry_id]
+    # Update coordinator in an async context.
+    await coordinator.update_more_info_url()
```

Comment:
Please grab this data in ipp/__init__.py, when the integration starts instead of the sensor.